ocaml/xapi/xapi_globs.ml

```diff
@@ -945,6 +947,8 @@ let repository_gpgkey_name = ref ""
 
 let repository_gpgcheck = ref true
 
+let bundle_repository_dir = ref "/var/xapi/bundle-repo/"
```

Comment:
remove the ending '/'

Reply:
Yes. A good reason to use `Filename.concat` later because it's not thrown off by such a path, unlike a simple string operation.